addons/pinniped/config-controller/config/rbac/role.yaml

```diff
@@ -0,0 +1,28 @@
+---
```

Comment:
So I'd rather redirec this to `./manifests`, and then move the `./deployment.yaml` file into manifests as well, to keep all of our deployment files in one place. 

Thoughts?

Reply:
I moved it.